packages/flutter_tools/lib/src/web/bootstrap.dart

```diff
@@ -196,6 +209,8 @@ let _scriptUrls = {
     ];
 
     let loadConfig = new window.\$dartLoader.LoadConfiguration();
+    // TODO(srujzs): Verify this is sufficient for Windows.
```

Comment:
Question for @bkonyi (maybe?): Do the tests here run on various platforms in the Flutter CI? Windows?

Reply:
Well, it turns out `forbidden_imports_test.dart` makes sure I don't import `dart:io`, so maybe I shouldn't go down that route. :) I'll keep a todo here to make this work for Windows if needed for now.

Reply:
Hmm, it looks like I should access `isWindows` using some of the abstractions we have in Flutter instead. Updated to use that.

Reply:
We definitely run the native targets using various platforms, but I'm not sure about web. Maybe @christopherfujino would know off the top of his head?

Reply:
not sure off the top of my head.